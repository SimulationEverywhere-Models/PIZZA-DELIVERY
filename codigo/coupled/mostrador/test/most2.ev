00:00:00:050 Abrir_m 1
00:00:05:000 Recibe_ent_m 2
00:00:15:000 Atiende_m 3
00:00:20:000 Recibe_ent_m 3
00:00:35:000 Cerrar_m 1
