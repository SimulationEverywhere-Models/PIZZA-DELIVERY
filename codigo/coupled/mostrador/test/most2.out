00:00:00:050 lista_ll_m 1
00:00:05:000 ocupada_ent_m 1
00:00:05:030 dist_out_m 2
00:00:05:030 libre_m_ent 1
00:00:15:000 ocupado_m 1
00:00:17:000 libre_m 1
00:00:17:030 manda_pedido 3
00:00:20:000 ocupada_ent_m 1
00:00:20:030 dist_out_m 3
00:00:20:030 libre_m_ent 1
00:00:35:000 cerrado_m 1
